src/main/java/io/vertx/ext/hawkular/impl/inventory/EntityReporter.java

```diff
@@ -95,7 +95,7 @@ protected static void createFeed(Future<Void> fut) {
 
   protected static void createResourceType(JsonObject body, Future<Void> fut) {
     HttpClientRequest request = httpClient.post(composeEntityUri("f;"+feedId, "resourceType"), response -> {
-      if (response.statusCode() == 201) {
+      if (response.statusCode() == 201 || response.statusCode() == 409) {
```

Comment:
Forgive my ignorance on Hawkular internals, but 4xx status codes usually mean a client error, so in this case a conflict (409) will be handled as resource created (201)? not knowing anything about the internals of Hawkular i'd assume that a 409 should be handled as an error, not as a success. Is this the intended behaviour?